charts/icm-as/values.yaml

```diff
@@ -72,6 +72,14 @@ datadog:
   # define (additional) datadog options
   additionalOptions:
 
+# configuration related to New Relic
+newrelic:
```

Comment:
Maybe consider also adding this section to the `values.yaml` in the `icm` umbrella chart.